packages/common/src/types/savedCharts.ts

```diff
@@ -6,9 +6,13 @@ export enum ChartType {
     BIG_NUMBER = 'big_number',
 }
 
-type BigNumberConfig = {
+export type BigNumber = {
+    label: string;
```

Comment:
I would make this optional so the user doesn't have to specify the label. It'll also make the migration much easier because we don't have to backfill all the existing configurations with a label